### DISCUSSION TITLE
MAPE vs sMAPE  in MeanAbsolutePercentageError( )

### DISCUSSION THREAD

**Discussion by k7e7n7t** [alan-turing-institute/sktime#1508] [Q&A]
I ran mean_absolute_percentage_error in cell 41 of 01-forecasting.ipynb and got 0.145427686270316.
I then ran MeanAbsolutePercentageError class with the default symmetric = False (which make it asymmetric just like MAPE I assume) but I get  0.13189432350948402 which I would expect from sMAPE, but not with the flag set to False.  If I set symmetric = True it does match.  This seems backwards to me.  What am I missing?

**Comment by fkiraly** [comment]
Yes, it seems like this is backwards? My naive user expectation would be MAPE default being `symmetric=False`.

@rnkuhns, FYI, have a look at this

**Comment by fkiraly** [comment]
@RNKuhns has offboarded and not replied to this.
I have moved this to issues as a bug report, so we can check it and deal with it there:

https://github.com/alan-turing-institute/sktime/issues/1986

**Reply by fkiraly** [reply]
proposed fix is in #2069, if you want to try out whether this agrees with your expectations, @k7e7n7t

**Reply by fkiraly** [reply]
this is now merged, @k7e7n7t - the fix is on the current development version, and will be in the next release (0.11.0).
